.github/workflows/certora.yml

```diff
@@ -0,0 +1,56 @@
+name: Certora
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+
+      matrix:
+        conf:
+          - DelegationEthreum
```

Comment:
```suggestion
          - DelegationEthreeum
```

Reply:
Fixed in https://github.com/morpho-org/morpho-token-upgradeable/pull/85/commits/bf1d4caac651a629435253001c0df22dfd8e16ac.